qds_sdk/qubole.py

```diff
@@ -87,3 +91,18 @@ def agent(cls, version=None):
           cls.cached_agent = Connection(cls._auth, cls.rest_url, cls.skip_ssl_cert_check)
 
         return cls.cached_agent
+
+    @classmethod
+    def get_cloud(cls, cloud=None):
```

Comment:
What if someone passed cloud = `"abc"` in this fn, we would return `cls.cloud_config` which is totally absurd.